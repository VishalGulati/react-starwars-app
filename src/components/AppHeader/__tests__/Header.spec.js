import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';

describe('Header', () => {
  it('should render Header properly', () => {
    const component = renderer.create(<Header />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
