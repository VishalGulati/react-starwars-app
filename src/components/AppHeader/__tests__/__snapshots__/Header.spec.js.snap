// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Header properly 1`] = `
<nav
  className="navbar navbar-default"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
        href="/"
      >
        <img
          alt="logo"
          className="logo"
          src="test-file-stub"
        />
      </a>
    </div>
    <ul
      className="nav navbar-nav"
      role={null}
    >
      <li
        className=""
        role="presentation"
        style={undefined}
      >
        <a
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
        >
          Star Wars
        </a>
      </li>
    </ul>
    <ul
      className="pull-right nav navbar-nav"
      role={null}
    >
      <li
        className=""
        role="presentation"
        style={undefined}
      >
        <a
          href="#"
          onClick={[Function]}
          onKeyDown={[Function]}
          role="button"
        >
          Logout
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
